tests/testthat/comparison_data/energy/L117.RsrcCurves_EJ_R_tradbio.csv

```diff
@@ -0,0 +1,60 @@
+# ../energy-data/level1//L117.RsrcCurves_EJ_R_tradbio.csv
+# Written by LA117.tradbio.R
+# Fri Mar 31 15:50:41 2017
+# " Traditional biomass resources by GCAM region "
+# " Unit = EJ "
```

Comment:
OK, thanks. I'm making a few tweaks (as it is still failing tests) but will then merge it.